lib/rules/disallow-newline-before-block-statements.js

```diff
@@ -97,7 +97,8 @@ module.exports.prototype = {
             var openingBrace = tokens[openingBracePos];
             var prevToken = tokens[openingBracePos - 1];
 
-            if (openingBrace.loc.start.line !== prevToken.loc.start.line) {
+            if (typeof prevToken === 'object' &&
```

Comment:
Nice catch & thanks for contributing!

Probably better to use here [new assertions](https://github.com/jscs-dev/node-jscs/blob/master/lib/token-assert.js#L81). Something like:
```
var openingBrace = file.getFirstNodeToken(node);
var prevToken = file.getPrevToken(openingBrace);
if (!prevToken) {
  return;
}

errors.assert.sameLine...
```